parsers/verilog.c

```diff
@@ -1630,6 +1633,18 @@ static int tagNameList (tokenInfo* token, int c)
 			if (c == '=')
 				c = skipExpression (c);
 		}
+		else if (c == '(' || c == '[') // should be instance
+		{
+			c = skipDimension(c);
+			c = skipPastMatch ("()");
+			c = skipWhite (c);
+			if (c == ';' || c == ',')
+			{
+				kind = K_INSTANCE;
```

Comment:
It seems that you missed my review comment this morning.
This variable is not necessary.  You can specify K_INSTANCE directly.

Reply:
Got it, thanks!